data/os/Debian/10.yaml

```diff
@@ -1,3 +1,4 @@
 ---
 letsencrypt::plugin::dns_rfc2136::package_name: 'python3-certbot-dns-rfc2136'
 letsencrypt::plugin::dns_route53::package_name: 'python3-certbot-dns-route53'
+letsencrypt::plugin::dns_cloudflare::package_name: 'python3-certbot-dns-cloudflare'
```

Comment:
This inspired me to submit https://github.com/voxpupuli/puppet-letsencrypt/pull/280

Reply:
@ekohl i added your idea https://github.com/voxpupuli/puppet-letsencrypt/pull/280 into this one here too.

Reply:
#280 is now merged, resulting in a merge conflict. Please rebase.

Reply:
Done